elastic/beats
# Include log.source.address for unparseable syslog messages

Continues with #13274, fixes #13268.

How to test:
* Start filebeat with the syslog input
* Send an invalid message to the open port
* Check that the generated event for the invalid message includes the source address

Co-authored-by: Brian Candler <b.candler@pobox.com>
